src/pages/Scholarships.jsx

```diff
@@ -42,7 +44,14 @@ function ScholarshipsPage() {
       <Typography variant="h3" component="h1" style={{ textAlign: 'center' }}>
         Scholarships
       </Typography>
-      <FilterBar />
+      <Grid container spacing={2}>
+        <Grid item sm={10}>
+          <FilterBar />
+        </Grid>
+        <Grid item sm={2}>
+          <SortBar />
```

Comment:
The sorting option dropdown is not really a bar, it's more _part_ of the filter bar. It might be better to move this (along with the Grid layout logic) to `FilterBar`. The filter options together would be one item of that grid, while the Sorting option be the other item of that grid (so that `space-between` maximizes the space between the two regions)

Reply:
Ok sure I moved it and kept the logic